src/os/ObjectStore.h

```diff
@@ -868,7 +868,7 @@ class ObjectStore {
           objects(t->object_index.size()) {
 
         ops = t->data.ops;
-        op_buffer_p = t->op_bl.get_contiguous(0, t->data.ops * sizeof(Op));
+        op_buffer_p = t->op_bl.c_str();
```

Comment:
I'm have a problem convincing myself that `op_bl` can store more than `ops * sizeof(Op)`, and thus `::get_contiguous()` brings any benefit over `::c_str()`.